index.ts

```diff
@@ -445,13 +445,17 @@ export const hasCode = (url: URL | Location = location): boolean => // Static co
 	isCompare(url) ||
 	isBlame(url);
 
-export const isActions = (url: URL | Location = location): boolean => url.pathname.startsWith('/marketplace/actions/');
-collect.set('isActions', [
+export const isMarketplaceAction = (url: URL | Location = location): boolean => url.pathname.startsWith('/marketplace/actions/');
+collect.set('isMarketplaceAction', [
 	'https://github.com/marketplace/actions/urlchecker-action',
 	'https://github.com/marketplace/actions/github-action-for-assignee-to-reviewer',
 	'https://github.com/marketplace/actions/hugo-actions',
 ]);
 
+/** @deprecated use isMarketplaceAction instead */
+export const isActions = isMarketplaceAction;
+collect.set('isActions', combinedTestOnly);
+
 /** @deprecated use isActions instead */
```

Comment:
This was supposed to be dropped in the last major

Reply:
D'oh!

Reply:
Ok let's clean this up and I'll release v4

Reply:
Merge this then I will do the other 2

Reply:
You can create a single Actions-related PR, it's easier